armi/reactor/tests/test_assemblies.py

```diff
@@ -1137,6 +1137,44 @@ def test_carestianCoordinates(self):
         )
         self.assertEqual(a.coords(), (2.0, 2.0))
 
+    def test_pinPlenumVolume(self):
+        """
+        Test the volume of a pin in the assembly's plenum.
+        """
+        assembly = assemblies.Assembly("dummy")
+        self.assertEqual(0.0, assembly.getPinPlenumVolumeInCubicMeters())
+
+        pinPlenumVolume = 7.853981633974482e-06
+        testAssemblies = buildTestAssemblies()
+        plenumBlock = makeTestAssembly(
+            1, 2, grids.CartesianGrid.fromRectangle(2.0, 2.0))
```

Comment:
I think `plenumBlock` in here is actually an `Assembly`, rather than a block. Get one of its child `Block`s by indexing into it.